Algorithm.Python/QuantConnect.Algorithm.PythonTools.pyproj

```diff
@@ -79,6 +80,7 @@
     <Compile Include="HourReverseSplitRegressionAlgorithm.py" />
     <Compile Include="HourSplitRegressionAlgorithm.py" />
     <Compile Include="IndicatorSuiteAlgorithm.py" />
+    <Compile Include="IndicatorWarmupAlgorithm.py" />
```

Comment:
wonder if there's a better way to do this. with the newer csproj format I believe you can just add an entire folder, for now we'll have to be diligent in remembering to add python algorithms in both places.